src/fugw/solvers/dense.py

```diff
@@ -121,11 +121,11 @@ def fugw_loss(
         pi1, pi2 = pi.sum(1), pi.sum(0)
         gamma1, gamma2 = gamma.sum(1), gamma.sum(0)
 
-        loss_wasserstein = None
-        loss_gromov_wasserstein = None
-        loss_marginal_constraint_dim1 = None
-        loss_marginal_constraint_dim2 = None
-        loss_regularization = None
+        loss_wasserstein = torch.zeros(1)
+        loss_gromov_wasserstein = torch.zeros(1)
+        loss_marginal_constraint_dim1 = torch.zeros(1)
+        loss_marginal_constraint_dim2 = torch.zeros(1)
+        loss_regularization = torch.zeros(1)
         loss = 0
```

Comment:
Out of curiosity, why not simply do `loss_wasserstein = 0`?

Reply:
This (I admit counter-intuitive) formulation is compatible with torch's ```.item()``` which previously failed when any loss remained at 0.
```
"wasserstein": loss_wasserstein.item(),
"gromov_wasserstein": loss_gromov_wasserstein.item(),
"marginal_constraint_dim1": loss_marginal_constraint_dim1.item(),
"marginal_constraint_dim2": loss_marginal_constraint_dim2.item(),
"regularization": loss_regularization.item(),
"total": loss.item(),
```

Reply:
Thanks for the quick answer!

Reply:
You're welcome ! 😊